### ISSUE TITLE
CSS changes refresh the entire page, instead of just the css file

### ISSUE THREAD

**Issue by mrchaarlie** [billybonks/ember-cli-stylelint#64]
Making a change to a `.scss` file will cause the entire page to refresh, instead of just updating the css file.  Ideally, the CSS will live-update and not cause the entire app to reload.

**Comment by billybonks** [comment]
Hey i am not sure if this is the scope of the linter. probably more of a ember-cli feature. unless that feature exists and the linter is breaking it ?

**Comment by mrchaarlie** [comment]
Yes, that is what I mean. Ember CLI by default only refreshes the CSS, but with the linter enabled, it for some reason refreshes the entire page.

**Comment by astronomersiva** [comment]
Can confirm this. Happens only with the style lint enabled.
